Regression: NoClassDefFoundError BaseLogger in PaConnectSupportTest on Wildfly

## What needs to be done?

Wildfly integration tests are failing with 
```
Error: 6,773 [ERROR] Errors: 
Error: 6,773 [ERROR]   PaConnectSupportTest » Deployment Cannot deploy test.war: {"WFLYCTL0062: Composite operation failed and was rolled back. Steps that failed:" => {"Operation step-1" => {"WFLYCTL0080: Failed services" => {"org.operaton.bpm.platform.process-application-module.\"test.war\".DEPLOY.pa" => "java.lang.NoClassDefFoundError: org/operaton/commons/logging/BaseLogger
    Caused by: java.lang.NoClassDefFoundError: org/operaton/commons/logging/BaseLogger
    Caused by: java.lang.ClassNotFoundException: org.operaton.commons.logging.BaseLogger from [Module \"org.operaton.bpm.operaton-engine-plugin-connect\" version 2.1.0-SNAPSHOT from local module loader @6a55299e (finder: local module finder @2f1de2d6 (roots: /home/runner/work/operaton/operaton/qa/wildfly-runtime/target/server/wildfly-38.0.1.Final/modules,/home/runner/work/operaton/operaton/qa/wildfly-runtime/target/server/wildfly-38.0.1.Final/modules/system/layers/base))]"}}}}
```

## Additional context

[Example log](https://github.com/operaton/operaton/actions/runs/24800404427/job/72585055593)

It is likely caused by the changes of https://github.com/operaton/operaton/pull/2768/changes

This introduced a dependency from the connect plugin to operaton-commons

--- a/distro/wildfly/modules/src/main/modules/org/operaton/bpm/operaton-engine-plugin-connect/main/module.xml
+++ b/distro/wildfly/modules/src/main/modules/org/operaton/bpm/operaton-engine-plugin-connect/main/module.xml
@@ -7,6 +7,7 @@
 
     <module name="java.base" />
 
+    <module name="org.operaton.commons.operaton-commons-logging" />
     <module name="org.operaton.bpm.operaton-engine" />
 
     <module name="org.operaton.connect.operaton-connect-core" />

--- a/distro/wildfly/modules/src/main/modules/org/operaton/bpm/operaton-engine/main/module.xml
+++ b/distro/wildfly/modules/src/main/modules/org/operaton/bpm/operaton-engine/main/module.xml
@@ -35,8 +35,8 @@
     <module name="org.operaton.bpm.model.operaton-cmmn-model" />
     <module name="org.operaton.bpm.model.operaton-dmn-model" />
 
-    <module name="org.operaton.commons.operaton-commons-logging" />
-    <module name="org.operaton.commons.operaton-commons-utils" />
+    <module name="org.operaton.commons.operaton-commons-logging" export="true"/>
+    <module name="org.operaton.commons.operaton-commons-utils" export="true"/>
     <module name="org.operaton.commons.operaton-commons-typed-values" export="true" />
     <module name="org.operaton.template-engines.operaton-template-engines-freemarker" services="import" />
     <module name="org.operaton.spin.operaton-spin-core" services="import"/>

